ddtrace/commands/ddtrace_run.py

```diff
@@ -107,11 +107,11 @@ def main():
     if os.path.basename(executable) == "uwsgi":
         print(
             (
-                "ddtrace-run is not supported with uwsgi. "
-                "See https://ddtrace.readthedocs.io/en/stable/advanced_usage.html#uwsgi for details."
+                "ddtrace-run is no longer supported with uWSGI since 0.48.0 release. "
```

Comment:
This implies that there was some change that was made in 0.48.0 that people might go looking for which breaks uWSGI compatibility (which there isn't, besides the fact that we add this message). Maybe something like 

> ddtrace-run has known compatibility issues with uWSGI that can result in workers failing to restart or other broken behaviour. It is recommended to remove ddtrace-run and update your uWSGI configuration following https://ddtrace.readthedocs.io/en/stable/advanced_usage.html#uwsgi

or similar